packages/manager/modules/pci/src/projects/project/components/activate-project-banner/activate-project-banner.controller.js

```diff
@@ -1,3 +1,19 @@
 export default class PciProjectActivateBannerController {
-  // Will be used for tracking functions
+  /* @ngInject */
+  constructor(atInternet) {
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.trackingName = `${this.trackingPageName}::activate-project`;
+
+    this.atInternet.trackPage({
+      name: this.trackingName,
+    });
+  }
+
+  onActivateProjectClick() {
+    this.atInternet.trackClick({ name: this.trackingName });
+    return this.goToDiscoveryProjectActivationPage();
```

Comment:
**nitpick**: 
### When the mobile view is displayed, this expression throws an error in the console because it is not bound to the component ###

Why do you use two different tracking methods in mobile and desktop versions ?

Reply:
The desktop & mobile views use the same function `onActivateProjectClick()` to track and redirect.